libraries/chain/block_header_state.cpp

```diff
@@ -463,24 +472,4 @@ namespace sysio { namespace chain {
       return std::get<protocol_feature_activation>(header_exts.lower_bound(protocol_feature_activation::extension_id())->second).protocol_features;
    }
 
-   block_header_state::block_header_state( legacy::snapshot_block_header_state_v2&& snapshot )
```

Comment:
We should look into why this is removed, I'm guessing it was a mismatch in the merge.

Reply:
IGNORE THIS, we won't have that version